packages/slice-machine/lib/models/common/widgets/RepeatableGroup/ListItem/index.jsx

```diff
@@ -0,0 +1,285 @@
+import { findWidgetByConfigOrType } from "@builders/utils";
```

Comment:
✍️ potential change

Can we not reuse ListItem component from the non-repeatable Group?

Reply:
Ah, I totally forgot to do that! Thank you for mentioning it.